### PR TITLE
Node environment

### PR DESCRIPTION
**Pull request by karljerico**
Added node environment.

- Created Dockerfile for Node
- Modify docker-compose.yml to add nodes

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -18,6 +18,21 @@ services:
...
+    build:
+      context: .
+      dockerfile: ./infrastracture/node/Dockerfile # Use the Dockerfile for Node.js
+    container_name: nodejs
+    volumes:
+      - ./src:/var/www/html  # Mount your Node.js application code
+    ports:
+      - 3000:3000  # Map container port to host port if needed
+    depends_on:
+      - backend  # Depend on your backend service or change as needed
```
Comment (hipe01mark) [inline docker-compose.yml]: Remove comments!

File: docker-compose.yml [hunk 2]
```diff
@@ -60,4 +75,5 @@ services:
     depends_on:
       - database
     networks:
-      - app
\ No newline at end of file
+      - app
+
```
Comment (bpocrafael) [inline docker-compose.yml]: trailling white space

File: infrastracture/node/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Use an official Node.js runtime as a parent image
```
Comment (bpocrafael) [inline infrastracture/node/Dockerfile]: too many unnecessary comments

**Review by hipe-francis — COMMENTED** [review]
LGTM, just note the previous reviews. 🚀

**Review by bpocrafael — APPROVED** [review]
LGTM 🚢 
 havent tested your docker but looks good to me

**Review by nadj1233 — APPROVED** [review]
LGTM 🐐

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

**Review by hipe-francis — APPROVED** [review]

**Pull request merged**